packages/embark/src/lib/modules/blockchain_process/proxy.js

```diff
@@ -123,30 +149,61 @@ class Proxy {
         this.commList[this.receipts[res.id]].blockNumber = res.result.blockNumber;
         this.commList[this.receipts[res.id]].gasUsed = res.result.gasUsed;
         this.commList[this.receipts[res.id]].status = res.result.status;
-        this.sendIpcMessage(this.commList[this.receipts[res.id]]);
+        this.contractLog(this.commList[this.receipts[res.id]]);
         delete this.transactions[this.commList[this.receipts[res.id]].transactionHash];
         delete this.commList[this.receipts[res.id]];
         delete this.receipts[res.id];
       }
+      // clear any tracked requests
+      const timeout = this.timeouts[res.id];
+      if(timeout) {
+        // if a dev tx had already been sent in an attempt to "push" this tx through,
+        // record that information.
+        if(timeout.txSent) {
+          const message = `✔ [Request ID ${res.id}]: Request successfully pushed through!\n` +
+          `Response details: ${JSON.stringify(res)}`;
+          this.consoleLog(`====================================================================\n${message}\n====================================================================`);
```

Comment:
Do we log those in the embark console, and is it for each request?

Reply:
In the embark console yes, but only when `loglevel` is `trace` and only when there are requests that don't get answered, or in the case when a request goes unanswered for a time and then gets a response later on (hasn't ever happened for me though).